reference/nstack_toolkit.rst

```diff
@@ -178,7 +178,44 @@ Used with the HTTP source, ``nstack send`` sends a JSON-encoded element to an en
 
 
 
+``test``
+^^^^^^^^
+
+.. code:: bash
+
+    $ nstack test <function> <input>
+
+=============    ===========
+Option           Description
+=============    ===========
+``function``     A fully-qualified function name, e.g. ``Foo:0.0.1.numChars``
```

Comment:
Yeah I realised this after I submitted this, I forgot that I'd changed the syntax (originally they were specified together) of the command and I wrote this for the old one. Whoops!